src/main.rs

```diff
@@ -48,6 +49,17 @@ fn setup(config_folder: &Path) -> IoResult<(Data<RwLock<Config>>, ServerConfig,
     };
     let config = Config::parse(&config_path).expect("failed to parse config");
     log::trace!("{:#?}", config);
+    if config.server.auth_token.is_some() {
+        log::warn!("[server].auth_token is deprecated, please use [server].auth_tokens");
+    }
+    if config.server.landing_page.is_some() {
+        log::warn!("[server].landing_page is deprecated, please use [landing_page].text");
+    }
+    if config.server.landing_page_content_type.is_some() {
+        log::warn!(
+            "[server].landing_page_content_type is deprecated, please use [landing_page].content_type"
+        );
+    }
```

Comment:
Maybe extract these into a new function called `warn_deprecation` or something like that. What do you think?

Reply:
Sure, but since this ciode will be deleted as soon as the deprecated stuff is removed, I didn't bother. Also I run into the same issue here as described here https://github.com/orhun/rustypaste/pull/84#discussion_r1264681653

But I'll look into it. If you have a quick answer as to what the type of config is, so that I can pass it into a function, I can do this in a few minutes. ;-)

Otherwise it will take a bit to figure this out. I also have a few other things planned for today so I might have to do that tomorrow.

Reply:
Hmm, I think it might be `Config`. At least the rust-analyzer says so. I'll run a few tests.

Reply:
Hmm, that was easier than expected. Done.